internal/scheduler/context/context.go

```diff
@@ -490,6 +512,44 @@ func (qctx *QueueSchedulingContext) ClearJobSpecs() {
 	}
 }
 
+// TotalCostForQueueWithAllocation returns the cost for which this queue should be penalised when computing fairness,
+// if the total allocation of this queue is given by allocated.
```

Comment:
This is a minor point, but I think that talking about "penalization" here (in addition to cost) is more confusing than anything.

```suggestion
// TotalCostForQueueWithAllocation returns the total cost of the resources allocated to this queue.
```